user.js

```diff
@@ -26,6 +26,134 @@ user_pref("browser.newtabpage.activity-stream.improvesearch.handoffToAwesomebar"
 // Browser Theme Based Scheme - Will be activate 95 Above
 // user_pref("layout.css.prefers-color-scheme.content-override", 3);
 
+// ** Theme Related Options ****************************************************
+// == Theme Distribution Settings ==============================================
+user_pref("userChrome.tab.connect_to_window",          true); // Original, Photon
```

Comment:
Would it be possible to have option names that are more distinguishable from builtins? (e.g. `black7375.lepton.userChrome.tab.connect_to_window`)

Reply:
I use `userChrome.` and `userContent` to keep it simple while distinguishing it from the built-in API.
If you search for it in `about:config` settings, it is an unused name.